sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets/README.md

```diff
@@ -0,0 +1,231 @@
+* [Delete a Secret](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#delete-a-secret)
+* [List Secrets](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-security-keyvault#list-secrets)
+* [Async create a Secret](https://github.com/samvaity/azure-sdk-for-python/tree/secrets-track2-readme/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets#async-create-a-secret)
+* [Async list Secrets](https://github.com/samvaity/azure-sdk-for-python/tree/secrets-track2-readme/sdk/keyvault/azure-security-keyvault/azure/security/keyvault/secrets#async-list-secrets)
+
+### Create a Secret
+`set_secret` creates a Secret to be stored in the Azure Key Vault. If a secret with the same name already exists then a new version of the secret is created.
+```python
+    secret = secret_client.set_secret("secret-name", "secret-value", enabled=True)
+
+    print(secret.name)
+    print(secret.value)
+    print(secret.version)
+    print(secret.enabled)
+```
+
+### Retrieve a Secret
+`get_secret` retrieves a secret previously stored in the Key Vault.
+```python
+    secret = secret_client.get_secret("secret-name")
+
+    print(secret.name)
+    print(secret.value)
+```
+
+### Update an existing Secret
+`update_secret` updates a secret previously stored in the Key Vault.
+```python
+    # Clients may specify the content type of a secret to assist in interpreting the secret data when it's retrieved
+    content_type = "text/plain"
+    # You can specify additional application-specific metadata in the form of tags.
+    tags = {"foo": "updated tag"}
+
+    updated_secret = secret_client.update_secret("secret-name", content_type=content_type, tags=tags)
+
+    print(updated_secret.name)
+    print(updated_secret.value)
+    print(updated_secret.version)
+    print(updated_secret.updated)
+    print(updated_secret.content_type)
+    print(updated_secret.tags)
+
+```
+
+### Delete a Secret
+`delete_secret` deletes a secret previously stored in the Key Vault. When [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/key-vault-ovw-soft-delete) is not enabled for the Key Vault, this operation permanently deletes the secret.
+```python
+    secret = secret_client.delete_secret("secret-name")
+
+    print(deleted_secret.name)
+    print(deleted_secret.deleted_date)
+```
+### List secrets
+This example lists all the secrets in the specified Key Vault.
+```python
+    secrets = secret_client.list_secrets()
+
+    for secret in secrets:
+        # the list doesn't include values or versions of the secrets
+        print(secret.name)
+```
+
+### Async operations
+Python’s [asyncio package](https://pypi.org/project/asyncio/) (introduced in Python 3.4) and its two keywords `async` and `await` serves to declare, build, execute, and manage asynchronous code.
```

Comment:
I'd remove the introduction to asyncio. I think it would be helpful here to mention that the async API is identical to the synchronous one, and that we support async on 3.5+.

Reply:
I'd keep the intro - it doesn't waste much space but adds clarity. Agreed about the second part